.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve Juju
+labels:
+  - kind/bug
+  - needs-triage
+body:
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: "Please describe the bug for us: "
+      placeholder: ex. "`juju status` returns the error `not found`"
+    validations:
+      required: true
+
+  - type: input
+    id: Version
+    attributes:
+      label: Juju version
+      description: "What version of Juju are you using?"
+      placeholder: ex. 3.6.1
+    validations:
```

Comment:
Can we add a regexp validator to ensure the user enters a valid version number

Reply:
+1 on that, I think it will be good so we limit bugs for 3.5 and 3.6

Reply:
Unfortunately there is no regex support yet.
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

Reply:
As long as in every automation we consume this field is protected. We don't need regex in the form, but rather to avoid generating wrong artifact (e.g. in Jira) due to a typo.